_sass/custom/_team.scss

```diff
@@ -1,41 +1,12 @@
 
 .team {
```

Comment:
Could you also add screenshots about the changes on Team? and all the pages affected by this change. (If none, you can just add the one screenshot for each, so we can be sure all are still good.

Reply:
Screenshots are now added for team and homepage